PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/repo/SampleData.java

```diff
@@ -22,6 +22,21 @@ public static void addSamplePlants(Map<Integer, Plant> repo) {
         repo.put(9, new Plant(9, "P. twoxl", "Pixel2XL", "This plant is rooted"));
     }
 
+    public static void addSamplePhotoPoints(List<PhotoPoint> repo) {
```

Comment:
Are you using these? If not, we can remove it.

Reply:
Yes, I will remove this, or more specifically embed it in the cardview, which is not yet attached to the database